docs/application/vscode-ext/web.md

```diff
@@ -0,0 +1,264 @@
+
+     > **Note**  
+     > If you click **NO**, it is judged that the installation has already been made and the project is created.
+
+## Install from a VSIX File
+
+To install VS Code Extension for Tizen Web from a VSIX file:
+
+1. In Visual Studio Code, at the bottom of the **Activity Bar**, click the **Extensions** icon.
+2. In the top right corner of the **Extensions** view, click the **More** button (![More icon](media/vscode_more_icon.png)) and select **Install from VSIX**.
+
+   ![Install from VSIX](media/tizen_web2.png)
+
+3. Select the VSIX file in the file browser and click **Install**.
+4. After the installation is complete, activate the extension by clicking **Reload**.
+
+   ![Install from VSIX](media/tizen_web3.png)
+
+5. Once the extension has been activated, Open the Command Palette and enter Create Wizard Project. It asks whether you want to use an existing Tizen baseline SDK installation or install a new one:
+
+   ![Baseline SDK installation query](media/tizen_web5.png)
+
+   - To perform a fresh installation, click **YES**.
+
+     Click **YES** to begin installing the Tizen baseline SDK. If it is already installed, skip the installation.
+
+     > **Note**  
+     > If you click **NO**, it is judged that the installation has already been made and the project is created.
+
+
+## Install Emulator Images
+
+If you do not have a real device, you can run applications in the Tizen emulator.
+
+To download emulator images, you can use the Tizen Package Manager or the Tizen Emulator Manager.
+
+- To use the Tizen Package Manager:
+  1. Open the **Command Palette** and enter **Run Package-Manager**.
+  2. Select the profiles and versions that you want to install and click **Install**.
+
+     ![Package Manager](media/tizen_package_manager.png)
+
+- To use the Tizen Emulator Manager:
+
+  > **Note**  
+  > The Tizen Emulator Manager shows the emulator images installation window only when no images are installed on your computer.
+
+  1. Open the **Command Palette** and enter **Run Emulator-Manager**.
+  2. Select the profiles and versions that you want to install and click **Ok**.
+
+     ![Emulator Manager](media/howtoinstall-emulatormanager.png)
+
+
+## Develop Applications
+
+The following sections explain how to use Visual Studio Code Extension for Tizen Web to develop your applications.
+
+### Create Tizen Web Project
+
+To create a Tizen Web project:
+
+1. Open the **Command Palette** and enter **Create Wizard Project**.
+
+   ![Create Wizard Project](media/tizen_web7.png)
+
+2. Select the type of project to be created.
+
+   ![Select the type of project](media/tizen_web8.png)
+
+3. Select the profile and version of the project to be created.
+
+   ![Select the profile and version](media/tizen_web9.png)
+
+4. Select the project to be created from the list.
+
+  > **Note**
+  > The list is different depending on the type and profile of the previously selected project.
```

Comment:
@sabeena21 

Yes. This is a necessary note.
This is because the contents may differ from the images depending on the previous selection.

Reply:
@DohyungLim, okay
Question: 
 "list is different",  please can you help me understand what does list refers to in this context?
Because, the below image is about selecting an application template?

Reply:
@sabeena21 
This is an image that means Templates.
A different list means it shows different templates.
Developers can fully understand.